flytectl/cmd/create/execution_util_test.go

```diff
@@ -48,6 +48,8 @@ func createExecutionUtilSetup() {
 
 func TestCreateExecutionForRelaunch(t *testing.T) {
 	s := setup()
+	defer s.RestoreStandardFileDescriptors()
```

Comment:
nit: maybe call this `teardown()` to match with go test name conventions, so users don't have to think about whether or not to call this function (and potentially miss doing so?)